### PR TITLE
chore(deps): make setup of project easier

### PR DESCRIPTION
**Pull request by jmcdo29**
Now with `yarn setup` and the updated `devDependencies` there are no more errors in the console about missing peer deps, and all packages _should_ be correcty linked, meaning minimal time to get the project up and running for new devs. @jeffminsungkim mind giving this a once over and double checking for me?

### CODE REVIEW

**Comment by jmcdo29** [comment]
Oh, also removes two `package-lock.json` files from previous builds of the packages.

File: package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
     "docker:socket": "yarn docker -t ogma/socket-test -f Docker/socket-io.Dockerfile .",
     "format": "prettier --write \"packages/**/*.ts\"",
     "lint": "eslint --ext .ts .",
+    "setup": "yarn && yarn build && yarn",
```
Comment (jeffminsungkim) [inline package.json]: Looks like `yarn && yarn build` should be enough.

**Review by jeffminsungkim — CHANGES_REQUESTED** [review]
I've left a comment.

I'm very concerned that both commands work fine after reloading the vscode. It's kinda annoying. 😨

**Review by ruslanguns — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (jmcdo29) [reply package.json]: Done

**Comment by jmcdo29** [comment]
I'm going to go ahead and merge these changes in. The problem about vscode is that the restart is necessary for the typescript server to understand the proper updates. Annoying, but I'm not sure if we have a way around it for now.

**Comment by ruslanguns** [comment]
ok. Testing the new monorepo branch!

**Comment by ruslanguns** [comment]
All tested and working smoothly!

**Pull request merged**